src/cli/args.js

```diff
@@ -1,5 +1,18 @@
 const parseArgs = () => {
-    // Write your code here 
+    let resStr = '';
+
+    process.argv.slice(2).forEach( (arg, index) => {
+
+        if (arg.slice(0,2) === '--') {
```

Comment:
This code could be simplified if you use 
`
for (let i = 0; i < process.argv.slice(2).length; i+=2) {
  // extraction logic goes here
}
`